src/components/MultiHopTrade/components/TradeInput/components/DonationCheckbox.tsx

```diff
@@ -32,7 +32,8 @@ export const DonationCheckbox: FC<DonationCheckboxProps> = memo(
     const isFromEvm = isEvmChainId(sellAsset.chainId)
     const affiliateBps = useAppSelector(selectActiveQuoteDonationBps)
     // disable EVM donations on KeepKey until https://github.com/shapeshift/web/issues/4518 is resolved
-    const showDonationOption = (walletIsKeepKey ? !isFromEvm : true) && affiliateBps !== undefined
+    const showDonationOption =
+      (walletIsKeepKey ? !isFromEvm : true) && affiliateBps !== undefined && affiliateBps !== '0'
```

Comment:
this was resulting in showing zero donations when bps was 0

Reply:
This breaks things a bit (I think it's this change that's doing it).

If you get a 0x quote that supports donations, and then uncheck donations, the donations checkbox permanently disappears and you can't re-enable it.

<img width="487" alt="Screenshot 2023-08-24 at 1 50 22 pm" src="https://github.com/shapeshift/web/assets/97164662/234a0e0f-3c56-4ca2-9feb-4a5c58ae5679">